src/account_manager/operations/stronghold_backup/mod.rs

```diff
@@ -0,0 +1,178 @@
+// Copyright 2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use std::{fs, path::PathBuf, sync::atomic::Ordering};
+
+use iota_client::secret::{stronghold::StrongholdSecretManager, SecretManager, SecretManagerDto};
+use zeroize::Zeroize;
+
+use crate::account_manager::{AccountHandle, AccountManager, AccountManagerBuilder};
+
+mod stronghold_snapshot;
+use stronghold_snapshot::{read_data_from_stronghold_snapshot, store_data_to_stronghold};
```

Comment:
prepend `self`

Reply:
Why?

Reply:
Because `cargo fmt` is not smart enough and without self it misclassify it as a 3rd party and puts it in the wrong import group